jobs/ci-kubernetes-e2e-kops-aws-updown.sh

```diff
@@ -51,26 +51,14 @@ export CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=true
 # Use default component update behavior
 export CLOUDSDK_EXPERIMENTAL_FAST_COMPONENT_UPDATE=false
 
-# AWS variables
-export KUBE_AWS_INSTANCE_PREFIX="${E2E_NAME}"
```

Comment:
Yeah, the kops config uses an explicit DNS hosted zone (in E2E_NAME) that it then populates.